web/src/Pages/Signup/Student.elm

```diff
@@ -377,41 +377,30 @@ viewDifficultyInfo model =
     if model.showDifficultyInfo then
         case model.signupParams.difficulty of
             "intermediate_mid" ->
-                [ Markdown.toHtml [ class "difficulty-info" ] """**Texts at the Intermediate Mid level** tend to be short public announcements,
-            selections from personal correspondence, clearly organized texts in very recognizable genres with clear
-            structure (like a biography, public opinion survey, etc.). Questions will focus on your ability to recognize
-            the main ideas of the text. Typically, students in second year Russian can attempt texts at this level. """
+                [ Markdown.toHtml [ class "difficulty-info" ] """**Texts at the Intermediate Mid level** tend to be short public announcements or 
+            very brief news reports that are clearly organized. Questions will focus on your ability to recognize the 
+            main ideas of the text. Typically, students in second-year Russian can attempt texts at this level. """
                 ]
 
             "intermediate_high" ->
-                [ Markdown.toHtml [ class "difficulty-info" ] """**Texts at the Intermediate High level** will tend to be several paragraphs in length,
-            touching on topics of personal and/or public interest.  The texts will tell a story, give a description or
-            explanation of something related to the topic. At the intermediate high level, you may be able to get the main
-            idea of the text, but the supporting details may be elusive. Typically, students in third year Russian can
-            attempt texts at this level."""
+                [ Markdown.toHtml [ class "difficulty-info" ] """**will tend to be several paragraphs in length, touching on topics of personal 
```

Comment:
Looks like `**Texts at the Intermediate Mid level** ` got lost here.

Reply:
Did you mean `**Texts at the Intermediate High level**` ?